src/main/java/org/myrobotlab/service/WebGui.java

```diff
@@ -313,11 +315,11 @@ public boolean allowExport(String serviceName) {
   }
 
   public void autoStartBrowser(boolean autoStartBrowser) {
-    this.autoStartBrowser = autoStartBrowser;
+    config.autoStartBrowser = autoStartBrowser;

```

Comment:
This means, handling and updating of the config is no longer limited to the same place in the classes...  configs can be updated really anywhere in the class... this is going to lead to places where the config was forgotten about.